src/components/NotificationBar/index.tsx

```diff
@@ -78,15 +87,26 @@ export default function NotificationBar() {
         </SIconButton>
       </SBadge>
       <SMenu
-        id="notificaion-menu"
+        id="notification-menu"
         anchorEl={anchorEl}
         open={open}
         onClose={handleClose}
         MenuListProps={{
-          'aria-labelledby': 'notificaion-menu-button',
+          'aria-labelledby': 'notification-menu-button',
         }}
       >
-        <MenuItem onClick={handleClose}>Notification 1</MenuItem>
+        {notifications
+          .filter((notification) => !notification.seen)
+          .map((notification) => (
+            <MenuItem
+              key={notification.id}
+              onClick={() => {
+                handleClose(notification);
+              }}
+            >
+              {notification.name}
+            </MenuItem>
+          ))}
```

Comment:
Please add `<MenuItem` with `No notifications' if there is nothing to show. 
Right now an emply drawer shows up